### PR TITLE
Added helper function to make strings possessive

### PR DESCRIPTION
**Pull request by jeffhappens**

### CODE REVIEW

File: src/Illuminate/Support/helpers.php [hunk 1]
```diff
@@ -639,6 +639,21 @@ function str_plural($value, $count = 2)
     }
 }
 
+if ( ! function_exists('str_possessive'))
```
Comment (GrahamCampbell) [inline src/Illuminate/Support/helpers.php]: cs

File: src/Illuminate/Support/helpers.php [hunk 2]
```diff
@@ -639,6 +639,21 @@ function str_plural($value, $count = 2)
...
+{
+    /**
+     * Make a string possessive
+     *
+     * @param  string  $string
+     * @return string|null
```
Comment (GrahamCampbell) [inline src/Illuminate/Support/helpers.php]: just string here

File: src/Illuminate/Support/helpers.php [hunk 3]
```diff
@@ -639,6 +639,21 @@ function str_plural($value, $count = 2)
     }
 }
 
+if ( ! function_exists('str_possessive'))
+{
+    /**
+     * Make a string possessive
```
Comment (GrahamCampbell) [inline src/Illuminate/Support/helpers.php]: missing fullstop

File: src/Illuminate/Support/helpers.php [hunk 4]
```diff
@@ -639,6 +639,21 @@ function str_plural($value, $count = 2)
...
+     *
+     * @param  string  $string
+     * @return string|null
+     */
+    function str_possessive($string)
+    {
+        return $string.'\''.($string[strlen($string) - 1] != 's' ? 's' : '');
+    }
+}
+
```
Comment (GrahamCampbell) [inline src/Illuminate/Support/helpers.php]: cs

**Comment by GrahamCampbell** [comment]
This doesn't appear to work at all?

**Comment by jeffhappens** [comment]
Sorry new to contributing.
It doesn't return anything?

**Comment by taylorotwell** [comment]
Going to hold off on this.

**Pull request closed without merging**